doc/api/crypto.md

```diff
@@ -2902,6 +2902,155 @@ is currently in use. Setting to true requires a FIPS build of Node.js.
 This property is deprecated. Please use `crypto.setFips()` and
 `crypto.getFips()` instead.
 
+### `crypto.argon2(password, salt, keylen[, options], callback)`
+
+<!-- YAML
+added: v21.0.0
```

Comment:
```suggestion
added: REPLACEME
```
The `REPLACEME` string will be replaced by the actual version by a releaser when included in a release.

Reply:
Fixed!